tiledb/sm/serialization/array.cc

```diff
@@ -200,6 +200,7 @@ Status array_from_capnp(
       for (auto array_schema_build : entries) {
         auto schema{array_schema_from_capnp(
             array_schema_build.getValue(), array->array_uri())};
+        schema.set_array_uri(array->array_uri());
```

Comment:
Can you add a TODO comment on those set_array_uri to say they should be removed when the #XXX ticket is fixed?